fiftyone/brain/internal/core/leaky_splits.py

```diff
@@ -1,108 +1,134 @@
 """
 Finds leaks between splits.
+
+| Copyright 2017-2024, Voxel51, Inc.
+| `voxel51.com <https://voxel51.com/>`_
+|
 """
+import logging
 
-from copy import copy
-import warnings
+import eta.core.utils as etau
 
-from fiftyone import ViewField as F
 import fiftyone.core.brain as fob
-import fiftyone.brain.similarity as sim
-import fiftyone.core.utils as fou
+import fiftyone.core.fields as fof
 import fiftyone.core.validation as fov
+import fiftyone.zoo as foz
+from fiftyone import ViewField as F
+
+import fiftyone.brain as fb
+import fiftyone.brain.similarity as fbs
+import fiftyone.brain.internal.core.utils as fbu
+
 
-fbu = fou.lazy_import("fiftyone.brain.internal.core.utils")
```

Comment:
Why is this done as a lazy import in similarity?

Reply:
The two main reasons `lazy_import()` is used are to avoid circular import errors and to avoid importing packages that aren't installed by default.

Can't remember why it was used in `fiftyone.brain.similarity` 🤷